src/types/fantom/fantom.ts

```diff
@@ -0,0 +1,235 @@
+import { CryptoNodeData, VersionDockerImage } from '../../interfaces/crypto-node';
+import { defaultDockerNetwork, NetworkType, NodeClient, NodeType, Role, Status } from '../../constants';
+import { Docker } from '../../util/docker';
+import { ChildProcess } from 'child_process';
+import { v4 as uuid} from 'uuid';
+import path from 'path';
+import os from 'os';
+import { Ethereum } from '../ethereum/ethereum';
+import { filterVersionsByNetworkType } from '../../util';
+import { FS } from '../../util/fs';
+import * as coreConfig from './config/core';
+
+
+export class Fantom extends Ethereum {
+
+  static versions(client: string, networkType: string): VersionDockerImage[] {
+    client = client || Fantom.clients[0];
+    let versions: VersionDockerImage[];
+    switch(client) {
+      case NodeClient.CORE:
+        versions = [
+          {
+            version: '1.1.0-rc.5',
+            clientVersion: '1.1.0-rc.5',
+            image: 'chainstack/fantom-opera:1.1.0-rc.5',
+            dataDir: '/root/.opera',
+            walletDir: '/root/keystore',
+            configDir: '/root/config',
+            networks: [NetworkType.MAINNET],
```

Comment:
Fantom does have a testnet and the image has the testnet genesis file in `/opt/genesis/testnet.g`. So, we will want to integrate testnet as well.